kornia/constants.py

```diff
@@ -61,3 +61,4 @@ class DataKey(ConstantBase, Enum, metaclass=EnumMetaFlags):
     BBOX_XYXY = 3
     BBOX_XYWH = 4
     KEYPOINTS = 5
+    TAG = 6
```

Comment:
Can you add a comment with a small definition of what TAG means ?

Reply:
TAG means the class labels for classification.

Reply:
not a very intuitive names compared to the others